src/widgets/current-refined-values/current-refined-values.js

```diff
@@ -36,8 +36,8 @@ const renderer = ({
   attributes,
   clearAllClick,
   clearAllURL,
-  clearRefinementClicks,
-  clearRefinementURLs,
+  clearRefinement,
```

Comment:
It's called `refine` too: https://github.com/algolia/react-instantsearch/blob/master/packages/react-instantsearch/src/components/CurrentRefinements.js